src/AsmResolver.DotNet/Cloning/MemberClonerListener.cs

```diff
@@ -0,0 +1,41 @@
+namespace AsmResolver.DotNet.Cloning
+{
+    /// <summary>
+    /// <see cref="MemberCloner"/> Callback Listener.
+    /// </summary>
+    public abstract class MemberClonerListener : IMemberClonerListener
+    {
+        /// <inheritdoc/>
+        public abstract void OnClonedMember(IMetadataMember original, IMetadataMember cloned);
```

Comment:
I think it makes sense to make these `virtual` rather than `abstract`. This makes it easier for consumers to use the base class, as they only have to override the methods they are interested in overriding. With everything abstract, consumers gain nothing over from just implementing the interface.